Sources/StreamChatUI/Common Views/CurrentChatUserAvatarView_Tests.swift

```diff
@@ -10,27 +10,39 @@ import XCTest
 class CurrentChatUserAvatarView_Tests: XCTestCase {
     var defaultAvatarView: CurrentChatUserAvatarView! = CurrentChatUserAvatarView().withoutAutoresizingMaskConstraints
     var customAvatarView: CustomCurrentChatUserAvatarView! = CustomCurrentChatUserAvatarView().withoutAutoresizingMaskConstraints
+    var defaultAppearanceHookAvatarView: CurrentChatUserAvatarView! = CurrentChatUserAvatarView().withoutAutoresizingMaskConstraints
```

Comment:
I don't think we need to store all these variants when they are valid just for one test.

Reply:
See other comments